examples/rust-outbound-pg/spin.toml

```diff
@@ -12,6 +12,6 @@ component = "outbound-pg"
 [component.outbound-pg]
 environment = { DB_URL = "host=localhost user=postgres dbname=spin_dev" }
 source = "target/wasm32-wasi/release/rust_outbound_pg.wasm"
-allowed_outbound_hosts = ["localhost:5432"]
+allowed_outbound_hosts = ["postgres://localhost:5432"]
```

Comment:
I'm not sure where this scheme comes from?  There's no scheme used in the connection string.

Reply:
We're not doing literal equality checking on strings: the scheme in this config signifies that you are allowed to make postgres outbound calls to that host and port (and not, for example, http calls).